lib/magento/version.rb

```diff
@@ -1,3 +1,3 @@
 module Magento
-  VERSION = '0.32.0'
+  VERSION = '0.32.1'

```

Comment:
Nitpick, but I'd up the minor version, not the patch version, since it's a new feature.